Tests/test_lib_pack.py

```diff
@@ -146,31 +146,34 @@ def test_HSV(self):
         self.assert_pack("HSV", "V", 1, (9,9,1), (9,9,2), (9,9,3))
 
     def test_I(self):
-        self.assert_pack("I", "I", 4, 0x04030201, 0x08070605)
         self.assert_pack("I", "I;16B", 2, 0x0102, 0x0304)
         self.assert_pack("I", "I;32S",
             b'\x83\x00\x00\x01\x01\x00\x00\x83',
             0x01000083, -2097151999)
 
         if sys.byteorder == 'little':
+            self.assert_pack("I", "I", 4, 0x04030201, 0x08070605)
```

Comment:
I think you missed moving the unpack version of these tests.

Reply:
Indeed. Done.